### DISCUSSION TITLE
Add support to favicons for other disk types

### DISCUSSION THREAD

**Discussion by SanderGo** [studio1902/statamic-peak-browser-appearance#5] [Ideas]
I've setup my favicons disk in filesystems to use s3 driver, but it looks like it's not utilizing it in the GenerateFavicons Listener class. It just grabs the svg with the public_path. 

So, I've made an update that uses Storage to get the correct path/url from the favicons disk config. 
Tested this with local storage and s3/DO. Works perfectly :)

Would you accept an PR for this?

Greets, Sander

**Comment by robdekort** [comment]
Sounds great, if it works and it helps you I'm in.

**Reply by SanderGo** [reply]
Cool, thanks Rob!
